acceptance/upgrade/upgrade_test.go

```diff
@@ -0,0 +1,78 @@
+package upgrade_test
+
+import (
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/epinio/epinio/acceptance/helpers/catalog"
+	"github.com/epinio/epinio/acceptance/helpers/proc"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Epinio upgrade with running app", func() {
+	var (
+		namespace string
+		appName   string
+	)
+
+	BeforeEach(func() {
+		namespace = catalog.NewNamespaceName()
+		env.SetupAndTargetNamespace(namespace)
+		appName = catalog.NewAppName()
+	})
+
+	AfterEach(func() {
+		env.DeleteApp(appName)
+		env.DeleteNamespace(namespace)
+	})
+
+	It("can upgrade epinio", func() {
+		domain := os.Getenv("EPINIO_SYSTEM_DOMAIN")
+		// Deploy a simple application before upgrading Epinio
+		out := env.MakeGolangApp(appName, 1, true)
+		routeRegexp := regexp.MustCompile(`https:\/\/.*omg.howdoi.website`)
+		route := string(routeRegexp.Find([]byte(out)))
+
+		// Check that the app is reachable
+		Eventually(func() int {
+			resp, err := env.Curl("GET", route, strings.NewReader(""))
+			Expect(err).ToNot(HaveOccurred())
+			return resp.StatusCode
+		}, 30*time.Second, 1*time.Second).Should(Equal(http.StatusOK))
+
+		// Build image with latest epinio-server binary
+		out, err := proc.Run("../..", false, "docker", "build", "-t", "epinio/epinio-server", "-f", "images/Dockerfile", ".")
```

Comment:
As `out` is not used you can use this syntax:
```
_, err := proc.Run("../..", false, "docker", "build", "-t", "epinio/epinio-server", "-f", "images/Dockerfile", ".")
Expect(err).NotTo(HaveOccurred())
```

Also `Expect(err).NotTo(HaveOccurred(), out)` can be replaced with `Expect(err).NotTo(HaveOccurred())`, but I don't know exactly the differences. But it works.

Reply:
If you don't provide a second parameter (`out`), when it fails you simply see the `err` object. Most of the time that doesn't tell you what was wrong (In my case it simply said `exit status 1`). You can pass the second parameter to add any useful text. In this case, `out` would be the output of the command which should be a lot more helpful.